content/_data/events/guadalajara-jam.adoc

```diff
@@ -0,0 +1,5 @@
+---
+title: "Guadalajara JAM"
+date: "2016-04-30T16:00:00"
```

Comment:
yeah..she changed the date and time recently. I will need make the edit in
the pull request.

On Thu, Apr 21, 2016 at 3:57 PM, Christopher Orr <notifications@github.com>
wrote:

> In content/_data/events/guadalajara-jam.adoc
> <https://github.com/jenkins-infra/jenkins.io/pull/232#discussion_r60668419>
> :
>
> > @@ -0,0 +1,5 @@
> > +---
> > +title: "Guadalajara JAM"
> > +date: "2016-04-30T16:00:00"
>
> The start time is listed as 18:45 on the Meetup page.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jenkins-infra/jenkins.io/pull/232/files/8f01e4bcd3b20689e64865ef8e6012d726210029#r60668419>
>